### PR TITLE
fix(docs): add level 2 swag to contribution guide

### PR DESCRIPTION
**Pull request by AnushPatel**
I've added a line to mention the Level 2 swag which was added recently. Please let me know what other additions can be made to make the Level 2 swag more 'highlighted' so to speak

### CODE REVIEW

**Comment by AnushPatel** [comment]
That is a Good idea @LekoArts! Don't know why it didn't come to me earlier

**Comment by AnushPatel** [comment]
What else should I add?

File: docs/docs/contributor-swag.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ To show our appreciation, _**everyone who contributes to Gatsby is eligible to r
 
 If you contribute to the Gatsby org on GitHub, you can claim your swag by [logging into the Gatsby Maintainer Dashboard](https://store.gatsbyjs.org/login) and requesting a discount code.
```
Comment (LekoArts) [inline docs/docs/contributor-swag.md]: Could you please change the link to `https://store.gatsbyjs.org`

File: docs/docs/contributor-swag.md [hunk 2]
```diff
@@ -12,6 +12,8 @@ To show our appreciation, _**everyone who contributes to Gatsby is eligible to r
...
 
+If you contribute five or more times, you can claim your _**Level 2**_ swag.
```
Comment (LekoArts) [inline docs/docs/contributor-swag.md]: Could you please add the sentence in the same paragraph, not as a new paragraph?
And I also like this sentence more:

`With five contributions or more you're eligible for Level 2 swag.`

Less repitition.

**Comment by AnushPatel** [comment]
Thank you @DSchau!

**Pull request merged**